CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [NEXT_RELEASE]
 ### Changed
 - Change Method `CreateSecret` to `CreateOrUpdateSecret` on k8s interfaces
+- Only deploy nginx if nginx.conf exist
+- Create configmap on deploy with nginx
 
```

Comment:
The feature hasn't been released yet so I think you shouldn't add these lines. What do you guys think?

Reply:
LGTM

Reply:
Agree